troubleshooting/faq.md

```diff
@@ -161,6 +162,41 @@ So it’s important to understand the different storage driver options available
 
 We recommend that you use the `overlay` driver, rather than `devicemapper`; for help configuring this [please see the following tutorial](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#configure-docker-with-the-overlayoverlay2-storage-driver)
 
+## What should I do if ssl problem occurs with npme over https?
+
+While working with custom certificate for npme registry, you might get an error related to ssl certificate on https while logging in or publishing a package, but it works fine on browser with https. It happens because node is a bit stricter about certificates than Chrome.
+
+To fix your issue temporarily you can provide `--strict-ssl=false` during login or you can add it to your ~/.npmrc file
+`npm config set strict-ssl=false`.
+
+For permanent fix you can point npm to a cafile
+```
+npm config set cafile /path/to/cert.pem
+```
+
+You can also configure ca string(s) directly.
+
+```
+npm config set ca "cert string"`
+```
+`ca` can be an array of cert strings too. In your .npmrc:
+
+```
+ca[ ]="cert 1 base64 string"
+ca[ ]="cert 2 base64 string"
+```
+
+The npm config commands above will persist the relevant config items to your ~/.npmrc file:
+
+```
+cafile=/path/to/cert.pem
+```
+
+
+Above CA settings will override the default "real world" certificate authority lookups that npm uses. If you try and use any public npm registries via https that aren't signed by your CA certificate, you will get errors.
```

Comment:
Minor correction: "The above cafile setting will override the default "real world" certificate authority lookups that npm uses."